### PR TITLE
QtumPrediction responsive

### PR DESCRIPTION
**Pull request by GoodThomas**

### CODE REVIEW

File: src/components/BottomBar/index.js [hunk 1]
```diff
@@ -49,33 +49,50 @@ export default class BottomBar extends Component {
...
-    </Typography>
+  <Grid container item xs={12} md={6} className={classes.bottomBarBlockInfoWrapper}>
+    <Grid item xs={12} sm={5}>
+      <Typography variant="body1" className={classes.bottomBarTxt}>
+        <span className={classes.bottomBarBlockNum}><FormattedMessage id="bottomBar.blockNum" defaultMessage="Current Block Number" />{`: ${blockNum}`}</span>
+      </Typography>
+    </Grid>
+    <Grid item xs={12} sm={7}>
+      <Typography variant="body1" className={classes.bottomBarTxt}>
+        <span className={classes.bottomBarBlockTime}><FormattedMessage id="bottomBar.blockTime" defaultMessage="Current Block Time" />: {blockTime ? getShortLocalDateTimeString(blockTime) : ''}</span>
```
Comment (dwalintukan) [inline src/components/BottomBar/index.js]: The `Current Block Time` should be on the same line at sizes > XS

image

File: src/scenes/QtumPrediction/components/TopActions/index.js [hunk 2]
```diff
@@ -13,53 +13,56 @@ import {
...
               variant="raised"
-              size="medium"
+              size={width === 'xs' ? 'small' : 'medium'}
               color="primary"
               className={classes.createEventButton}
               onClick={createEvent.open}
             >
-              <AddIcon fontSize={fontSize} />
+              <AddIcon className={classes.createEventButtonIcon} fontSize={fontSize} />
               <FormattedMessage id="create.dialogTitle" defaultMessage="CREATE AN EVENT" />
```
Comment (dwalintukan) [inline src/scenes/QtumPrediction/components/TopActions/index.js]: This text is missing on XS size:
image

Comment (dwalintukan) [reply src/scenes/QtumPrediction/components/TopActions/index.js]: It's also missing the `Sort By:` text.

Comment (GoodThomas) [reply src/scenes/QtumPrediction/components/TopActions/index.js]: image

File: src/components/BottomBar/index.js (hunk 1 above)
Comment (GoodThomas) [reply src/components/BottomBar/index.js]: I developed in Chinese before, not paid attention to English. I'll pay more attention to English next time.

File: src/scenes/QtumPrediction/components/TopActions/index.js (hunk 2 above)
Comment (GoodThomas) [reply src/scenes/QtumPrediction/components/TopActions/index.js]: Why is it different from what I see?

Comment (dwalintukan) [reply src/scenes/QtumPrediction/components/TopActions/index.js]: was seeing an old cached version sorry.

File: src/components/BottomBar/index.js (hunk 1 above)
Comment (dwalintukan) [reply src/components/BottomBar/index.js]: Yes please. Make sure all the changes look good in all languages.

**Pull request merged**